ccd-definition/CaseEvent/CareSupervision/MultiState.json

```diff
@@ -252,8 +252,8 @@
     "PreConditionState(s)": "Open;RETURNED",
     "PostConditionState": "*",
     "SecurityClassification": "Public",
-    "ShowSummary": "N",
-    "ShowEventNotes": "N",
+    "ShowSummary": "Y",
+    "ShowEventNotes": "Y",
```

Comment:
Do we want to change the event notes as part of this?

Reply:
rather no

Reply:
no mistake :)